ballot/models.py

```diff
@@ -52,6 +52,10 @@ def __str__(self):
 class BallotItemResponse(models.Model):
     """
     YES/NO to a referendum, or a candidate.
+
+    This is a *conceptually* abstract class, but exists
+    as a Django model so that finance can point to propositions
+    and candidates equally.
     """
```

Comment:
Looks like the latest spec changed things a bit; it's called `BallotSelection` now: 
https://github.com/votinginfoproject/vip-specification/blob/vip51/vip_spec.xsd#L180